pkgs/development/tools/yarn2nix-moretea/yarn2nix/lib/urlToName.js

```diff
@@ -20,9 +21,15 @@ function urlToName(url) {
     return path.basename(url);
   }
 
-  return url
+  let name = url
     .replace(/https:\/\/(.)*(.com)\//g, "") // prevents having long directory names
     .replace(/[@/%:-]/g, "_"); // replace @ and : and - and % characters with underscore
+
+  if (!(name.endsWith(".tar.gz") || name.endsWith(".tgz"))) {
+    name = name + ".tgz";
+  }
```

Comment:
Apologies for taking so long to properly review this

What was the motivation for this change?

It's backwards in-compatible with the `fetchYarnDeps` FOD fetcher iirc